src/sentry/integrations/jira_server/integration.py

```diff
@@ -244,11 +246,35 @@ def get_link_issue_config(self, group, **kwargs):
             if field["name"] == "externalIssue":
                 field["url"] = autocomplete_url
                 field["type"] = "select"
+
+        default_comment = "Linked Sentry Issue: [{}|{}]".format(
+            group.qualified_short_id,
+            absolute_uri(group.get_absolute_url(params={"referrer": "jira_server_integration"})),
```

Comment:
@earshinov I know I didn't give any guidance but can you change the referrer to just `jira_server`?

Reply:
Sure, fixed in 0ba1598ae556b1636aaef0d1c17c17f330af0f4b